modules/acs-architecture-overview.adoc

```diff
@@ -80,7 +80,7 @@ You deploy the per-cluster components into each cluster that you want to monitor
 [discrete]
 === Sensor
 
-{product-title} uses the Sensor component to monitor Kubernetes and OpenShfit clusters.
+{product-title} uses the Sensor component to monitor Kubernetes and OpenShift clusters.
```

Comment:
If there's nothing peculiar about RHACS that would break this, it should be:

```suggestion
{product-title} uses the Sensor component to monitor Kubernetes and {product-title} clusters.
```